x/blobstream/types/validator.go

```diff
@@ -112,30 +114,35 @@ func EVMAddrLessThan(e common.Address, o common.Address) bool {
 // increases by 1% due to inflation, we shouldn't have to generate a new
 // validator set, after all the validators retained their relative percentages
 // during inflation and normalized Blobstream power shows no difference.
-func (ibv InternalBridgeValidators) PowerDiff(c InternalBridgeValidators) float64 {
-	powers := map[string]int64{}
+func (ibv InternalBridgeValidators) PowerDiff(c InternalBridgeValidators) sdk.Dec {
+	powers := map[string]sdk.Dec{}
 	// loop over ibv and initialize the map with their powers
 	for _, bv := range ibv {
-		powers[bv.EVMAddress.Hex()] = int64(bv.Power)
+		powers[bv.EVMAddress.Hex()] = sdk.NewDec(int64(bv.Power))
```

Comment:
we can avoid this unsafe conversion using `sdk.NewDecFromBigInt(new(big.Int).SetUint64(bv.Power))`

Reply:
Thanks, done https://github.com/celestiaorg/celestia-app/pull/2719/commits/efae96232a7187b32179e37e298fddb4348b15b1.

Reply:
Actually this commit is the correct one https://github.com/celestiaorg/celestia-app/pull/2719/commits/bddfcfa3823718def693ee3293bb57183d0512d7